ui/ui-frontend/projects/vitamui-library/src/lib/components/reclassification-dialog/reclassification-dialog.component.ts

```diff
@@ -94,89 +94,191 @@ export enum ReclassificationToggle {
     CdkStepperModule,
     StepperComponent,
     ReactiveFormsModule,
-    NgIf,
     MatProgressSpinner,
     BadgeComponent,
     TranslatePipe,
     I18nPluralPipe,
     SelectComponent,
     MatButtonToggleGroup,
     MatButtonToggle,
-    NgForOf,
     FilingPlanModule,
     MatDialogContent,
     MatDialogActions,
     NextStepComponent,
     PreviousStepComponent,
     InputComponent,
   ],
+  providers: [BaseReclassificationDialogService],
 })
-export class ReclassificationDialogComponent implements OnInit, OnDestroy {
+export class ReclassificationDialogComponent implements OnInit, AfterViewInit {
+  @ViewChild('stepper') stepperRef!: StepperComponent;
```

Comment:
On doit pouvoir remplacer par
```suggestion
  stepperRef = viewChild.required(StepperComponent);
```